packages/components/src/theme/types.ts

```diff
@@ -0,0 +1 @@
+export const SIZES = ['xs', 'sm', 'md', 'lg'] as const;
```

Comment:
Could we remove this and use the same pattern we do for other components? Check Button as a reference